tests/test_core.py

```diff
@@ -7046,7 +7046,7 @@ def encode_utf8(data):
         # to the 'sources' field.
         # TODO(jgravelle): when LLD is the wasm-backend default, make sure it
         # emits only the files we have lines for.
-        assert len(data['sources']) > 1, data['sources']
+        assert len(data['sources']) > 0, data['sources']
```

Comment:
Is this only emitting one file now? If so we should get rid of this else branch entirely.

Reply:
OIC, Inverted this since its only s2wasm that is weird.

Reply:
Ah, right. Yeah that's much more clear.